Games/types/Mafia/roles/cards/DonateLife.js

```diff
@@ -0,0 +1,39 @@
+const Card = require("../../Card");
+const { PRIORITY_NIGHT_SAVER } = require("../../const/Priority");
+
+module.exports = class DonateLife extends Card {
+  constructor(role) {
+    super(role);
+
+    this.meetings = {
+      "Harvest Life": {
+        states: ["Night"],
+        flags: ["voting"],
+        targets: { include: ["alive"], exclude: ["dead", "self"] },
+        action: {
+          priority: PRIORITY_NIGHT_SAVER - 1,
+          run: function () {
+            if (!this.player.alive) {
```

Comment:
wont work, people only die at priority 0

Reply:
the simplest way to fix this is to give the extra life after the death
i.e. extra life does not work tonight

Reply:
fixed, also changed second action to an effect_giver priority